zio-http/shared/src/main/scala/zio/http/multipart/mixed/Mixed.scala

```diff
@@ -0,0 +1,304 @@
+package zio.http.multipart.mixed
```

Comment:
zio.http

Reply:
do u mean move it to the `zio.http` package, in this case I guess it should be renamed to `MultipartMixed`
right?